src/sections/Home/Layer5-statement/statement.style.js

```diff
@@ -50,7 +50,7 @@ const BannerSectionWrapper = styled.section`
     .hero {
         background: url(${imgHero});
         right: 0px;
-        height: 850px;
+        height: 740px;
```

Comment:
Use `rem` instead

Reply:
`37rem` to be exact.

Reply:
And `vertical-align: middle`

Reply:
Also, for `.left-child` class on line: 18 update CSS to
``` .left-child {
        position: relative;
        align-self: center;
```

Reply:
Also, the image height is not responsive around 950px and less, so please fix that as well.

Reply:
okay